lib/rules/jsx-closing-bracket-atomic.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @fileoverview Ensures the "/>" and "</" closing indicators are written as a single token
+ * @author Diogo Franco (Kovensky)
+ */
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Constants
+// ------------------------------------------------------------------------------
+
+var MESSAGE = 'JSX tag closing has embedded whitespace';
```

Comment:
I don't check the position of the tag name here, only for the presence of `/>` and `</`.

Thinking about it, checking for the tag name sounds like a job for a possible "jsx-space-after-opening" rule...